src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo/PostInfo';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
```

Comment:
it's better to use semantic tags for list, i.e `ul` (the same for items)
but don't forget to fix styles